boards/xtensa/esp32/Makefile.board

```diff
@@ -1,4 +1,3 @@
-# Default to xt-run but let the user override to xtsc-run if they desire
-# TODO
 EMU_PLATFORM ?=
 DEBUG_SCRIPT :=
+FLASH_SCRIPT := esp32.sh
```

Comment:
do you expect esp32.sh to do more than flash?  if not might rename script to esp32-flash.sh